integration_tests/src/main/python/orc_test.py

```diff
@@ -82,9 +82,16 @@ def test_basic_read(std_input_path, name, read_func, v1_enabled_list, orc_impl,
     StructGen([['child0', byte_gen], ['child1', orc_basic_struct_gen]]),
     StructGen([['child0', ArrayGen(short_gen)], ['child1', double_gen]])]
 
+all_basic_map_gens = [MapGen(f(nullable=False), f()) for f in [BooleanGen, ByteGen, ShortGen, IntegerGen, LongGen, FloatGen, DoubleGen, TimestampGen]] + [simple_string_to_string_map_gen]
+all_basic_map_gens += [DateGen(start=date(1590, 1, 1))]
```

Comment:
seems this is not a Map type?

Reply:
The same question. `DateGen ` is obviously not a `MapGen`. You may refer to the `MapGen ` for Parquet reader tests here https://github.com/NVIDIA/spark-rapids/blob/branch-21.10/integration_tests/src/main/python/parquet_test.py#L45

Reply:
fixed, referred parquet map gen